build/defines.mak

```diff
@@ -8,8 +8,8 @@ TOX_INI := $(OUTPUT_DIR)/tox.ini
 DRIVER_DIR := $(ROOT_DIR)/src/$(DRIVER)
 METADATA_DIR := $(DRIVER_DIR)/metadata
 METADATA_FILES := $(wildcard $(METADATA_DIR)/*.py)
-COMMON_METADATA_DIR := $(ROOT_DIR)/src/common/metadata
-METADATA_DIRS := $(METADATA_DIR) $(COMMON_METADATA_DIR)
+SHARED_PROTOS_DIR := $(ROOT_DIR)/src/shared_protos
+METADATA_DIRS := $(METADATA_DIR) $(SHARED_PROTOS_DIR)
```

Comment:
It's a little weird having both a METADATA_DIR variable and a METADATA_DIRS variable.

Reply:
Renamed to PROTO_DIRS, since we use it to look for .proto files.